pkg/cephmgr/osd/scheme.go

```diff
@@ -26,21 +26,29 @@ import (
 )
 
 const (
-	WalPartitionName      = "wal"
-	DatabasePartitionName = "db"
-	BlockPartitionName    = "block"
-	walPartition          = 1
-	databasePartition     = 2
-	blockPartition        = 3
-	UseRemainingSpace     = -1
-	schemeFilename        = "partition-scheme"
-	WalDefaultSizeMB      = 576
-	DBDefaultSizeMB       = 20480
+	Filestore                     = "filestore"
```

Comment:
I struggled on this @kokhang.  Golang support for enums is a little odd (idiomatic go uses an int to represent them) and I don't think it plays well with values that a user is expected to type into yaml/json.  Since the user is supposed to specify store type in the cluster TPR yaml, I don't know how to represent that with an enum,  allow them to type in a nice string and still get kubernetes to marshall it into the nice strongly typed golang ClusterSpec struct.  I believe that using consts for the values is OK so that the user can use a string easily in the cluster TPR that they type up.